fdbserver/RestoreMaster.actor.cpp

```diff
@@ -345,6 +366,7 @@ void dummySampleWorkload(Reference<RestoreMasterData> self) {
 		} else {
 			self->rangeToApplier[StringRef(keyrangeSplitter[i].toString())] = applier.first;
 		}
+		i++;
```

Comment:
This can be moved insider `i == 0` block, so it won't be run for the `else` case.

Reply:
I want to assign different [i] to `rangeToApplier`